### DISCUSSION TITLE
Why does the default configuration behave inconsistently under macos and under Linux?

### DISCUSSION THREAD

**Discussion by cathaysia** [wezterm/wezterm#3842] [Q&A]
The behavior of the same file is obviously inconsistent under macos and under linux. These inconsistencies don't just cover some system-related behavior.

For example, under linux, the new tab page is always in /home/xxx/, while under macos, it is in the current path. Under linux, you can freely switch between words with the left mouse button, but under macos, you can only select words, not characters.

**Comment by nekowinston** [comment]
That sounds like your Linux install is missing the [Shell Integration](https://wezfurlong.org/wezterm/shell-integration.html). (https://github.com/wez/wezterm/tree/main/assets/shell-integration)

**Comment by cathaysia** [comment]
Is it possible to make wezterm on macos still follow the behavior on linux. For example, ensure that the physical keys of the shortcut keys are consistent

**Reply by nekowinston** [reply]
Yes, you can simply disable all defaults and set every shortcut yourself with `wezterm show-keys --lua` and `config.disable_default_key_bindings = true`.
https://wezfurlong.org/wezterm/config/default-keys.html?h=disable_default_key_bindings

**Reply by cathaysia** [reply]
thanks, I'll check and test it. :)

**Comment by cathaysia** [comment]
Is it possible to make wezterm on macos still follow the behavior on linux. For example, ensure that the physical keys of the shortcut keys are consistent
